Reflection_Engine/Convert/ToText.cs

```diff
@@ -35,6 +35,19 @@ public static partial class Convert
         /**** Public Methods                            ****/
         /***************************************************/
 
+        public static string ToText(this MemberInfo member, bool includePath = false)
```

Comment:
I get that `MemberInfo` is neither acting on an `interface`nor is receiving a `BHoMObject`, but I think the pattern is still the one of the interface method. Is there any reason why we are not using that pattern?

Reply:
I considered it but 
- MemberInfo is not an interface. 
- We are not dealing with BHoMObjects as you said so I didn't feel like the pattern was relevant here as you would have to know the details of the MemberInfo class to even consider using the IToText method instead. So I felt more intuitive to no ask the user to choose between IToText and ToText 
- It is not actually a dynamic call to another method as it can also handle a default case on its own. Admittedly, we could reformat the code to change that but this keeps things nice and contained.